README.rst

```diff
@@ -43,6 +43,53 @@ To install from github::
 
     $ pip install git+git://github.com/robinedwards/neomodel.git@HEAD#egg=neomodel-dev
 
+Authentication
+--------------
+Please note that if you are utilizing Neo4j version 2.2 or newer there are
+some additional setup steps necessary relating to authentication. As of version 2.2
+Neo4j authentication is activated by default on new instances. Please follow the
+outstanding documentation provided by py2neo's Authentication_
+section to setup new credentials. If you are utilizing a hosted service this
+is most likely already taken care for you.
+
+.. _Authentication: http://py2neo.org/2.0/essentials.html#authentication
```

Comment:
I am just wondering if we need to cover how to set the username and password from neomodel in the getting started section: https://github.com/robinedwards/neomodel/blob/master/doc/source/getting_started.rst